app/views/users/totp_setup/new.html.erb

```diff
@@ -39,14 +39,14 @@
           <code class="grid-col-fill font-family-mono padding-y-2 padding-x-1 border-base-lighter border-1px text-bold" id="qr-code">
             <%= @code %>
           </code>
-          <%= render ClipboardButtonComponent.new(
-                clipboard_text: @code.upcase,
-                outline: true,
-                class: 'margin-right-0 margin-top-2 mobile-lg:margin-top-0 mobile-lg:margin-left-2 ico ico-copy',
-              ) do %>
-            <%= t('links.copy') %>
-          <% end %>
         </div>
```

Comment:
Will give a closer look shortly, but: Do we need this `div` anymore? One thing I notice is it was applying `flex-align-center`, which I assume was to vertically center the `code` and button when they were part of the same line. Since they aren't anymore, probably could at least remove that class.

Reply:
Yeah, looking again, I think we could get rid of a lot of the grid stuff that existed previously only to support the horizontal arrangement.

The `div` could make sense to keep, since `code` is an inline element, or we could just apply `display: block` to it.

```diff
diff --git a/app/views/users/totp_setup/new.html.erb b/app/views/users/totp_setup/new.html.erb
index 673d99d9e..a6bd05d11 100644
--- a/app/views/users/totp_setup/new.html.erb
+++ b/app/views/users/totp_setup/new.html.erb
@@ -35,15 +35,13 @@
           <%= image_tag @qrcode, skip_pipeline: true, alt: t('image_description.totp_qrcode') %>
         </div>
         <%= t('instructions.mfa.authenticator.manual_entry') %>
-        <div class="grid-row margin-top-2 flex-align-center">
-          <code class="grid-col-fill font-family-mono padding-y-2 padding-x-1 border-base-lighter border-1px text-bold" id="qr-code">
-            <%= @code %>
-          </code>
-        </div>
+        <code class="display-block margin-y-2 font-family-mono padding-y-2 padding-x-1 border-base-lighter border-1px text-bold" id="qr-code">
+          <%= @code %>
+        </code>
         <%= render ClipboardButtonComponent.new(
               clipboard_text: @code.upcase,
               outline: true,
-              class: 'ico ico-copy margin-top-2',
+              class: 'ico ico-copy',
             ) do %>
           <%= t('links.copy') %>
         <% end %>
```

Reply:
love it, thanks! added in 2954f6631f41b3702cd1f49c1eb4f703a672e071